# This message transports the result of image analysis.

uint8 bed_id
uint8 bed_side

uint32 fruit_sum
uint32 fruit_left
uint32 fruit_right